src/Header/SetCookie.php

```diff
@@ -39,9 +39,9 @@ class SetCookie implements MultipleHeaderInterface
      * @internal
      */
     const SAME_SITE_ALLOWED_VALUES = [
-        self::SAME_SITE_STRICT,
-        self::SAME_SITE_LAX,
-        self::SAME_SITE_NONE,
+        'strict',
+        'lax',
+        'none',
```

Comment:
I would suggest it to be changed to
```php
 const SAME_SITE_ALLOWED_VALUES = [
        'strict' = self::SAME_SITE_STRICT,
        'lax' = self::SAME_SITE_LAX,
        'none' = self::SAME_SITE_NONE,
    ];
```
Lowercase assertion can be made against key and additionally that will allow to set canonical form value:
```
$this->sameSite = self::SAME_SITE_ALLOWED_VALUES[strtolower($sameSite)];
```

Reply:
Considering that users might check value against constant, please make that change and add additional test ensuring case insensitive value is set to canonical form.